docs/application/tizen-studio/release-notes/2-4-release-notes.md

```diff
@@ -0,0 +1,86 @@
+# Tizen Studio 2.4 Release Notes
+
+
+## IDE and Tools
+
+
+### New Features
+
+-   Device Manager
+	-   Enable Device Manager Context UI to be extended by plugins
+-   RT-IDE
+	-   RT-IDE GCC Toolchain update to 6.3.1
+-   SmartThings
+	-   Make setupId mandatory field and map it to device onboarding id
+	-   Updated ramls for smart things resources
+-   Package Manager
+	-   Remove *.po files getting packaged into tpk
+
+### Changed Features
+-	RT-IDE
+	-   Removed local template for creating project. Project can be created using latest RT code from github.
+
+### Fixed Bugs
+-   Device Manager
+	-   Fix crash when filtering devices in JAVA 9
+	-   "Add new tab" button clickable area changed to full button
+	-   Added support for empty filter category in logs
+	-   Fixed Permit to install failure for secure TV profile
+-   Emulator
+	-   Corrected the marked location data in emulator control panel
+-   Package Manager
```

Comment:
Package Manager

-     Added https support in Package Manager to download SDK from https website.
-     Fixed multiple instance of same Package Manager getting launched in Windows and macOS.